src/main/proto/core/usubscription/v2/usubscription.proto

```diff
@@ -311,7 +311,7 @@ message DeprecateTopicRequest {
 // any uE that called RegisterForNotifications().
 message Update {
   // List of Subscription records
-  Topic topic = 1;
+  UUri topic = 1;
```

Comment:
I think the comment might be incorrect. something does not make sense

Reply:
fixed thank you